packages/daemon/package.json

```diff
@@ -32,20 +32,26 @@
     "@types/mysql": "^2.15.21",
     "@types/node": "^17.0.45",
     "@types/ws": "^8.5.5",
-    "@typescript-eslint/eslint-plugin": "^6.7.3",
-    "@typescript-eslint/parser": "^6.7.3",
+    "@typescript-eslint/eslint-plugin": "^7.4.0",
```

Comment:
Why did you upgrade this major version of this package? Shouldn't we do any upgrades in a separate PR to be better reviewed?

Reply:
Reverted all package changes related to eslint, I'll do it in this PR:
https://github.com/HathorNetwork/hathor-wallet-service/pull/156